packages/angular/projects/ui-angular/src/lib/primitives/button/button.component.ts

```diff
@@ -7,6 +7,7 @@ import { Component, HostBinding, Input, OnInit } from '@angular/core';
 export class ButtonComponent implements OnInit {
   @Input() type: 'submit' | 'button' = 'button';
   @Input() fullWidth: boolean | string = false;
+  @Input() disabled: boolean | string = false;
```

Comment:
To match our naming conventions with React counterparts, let's name this to `isDisabled`. https://ui.docs.amplify.aws/react/components/button

Reply:
Done with this changes.